src/lib/components/navbar/Navbar.component.js

```diff
@@ -48,6 +55,26 @@ const NavbarMenu = styled.div`
   align-items: center;
 `;
 
+const NavbarTabs = styled.div`
+flex: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const TabItems = styled.div`
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;  
+  margin: 0 1vh;
+  :hover{
+    height: 100%;
+    border-bottom: 1vh solid black;
```

Comment:
The border-bottom should be inside the navbar ;) like below by adding `box-sizing: border-box;`
![image](https://user-images.githubusercontent.com/47394132/68412437-7f3d6280-018c-11ea-8045-175618ccf4c1.png)

and make sure that all the item are aligned vertically.

Reply:
Regarding color, please use `color: ${mergeTheme(props.theme, defaultTheme).primary};` for the label and the border  to make the component themeable

Reply:
okay :+1: